examples/platform/efr32/wf200/host_if.cpp

```diff
@@ -488,679 +495,795 @@ static void sl_wfx_client_connected_callback(uint8_t * mac)
  *****************************************************************************/
 static void sl_wfx_ap_client_rejected_callback(uint32_t status, uint8_t * mac)
 {
-    {
-        // TODO
-        EFR32_LOG("Client rejected, reason: %d, MAC: %02X:%02X:%02X:%02X:%02X:%02X\r\n", (int) status, mac[0], mac[1], mac[2],
-                  mac[3], mac[4], mac[5]);
-    }
+    EFR32_LOG("Client rejected, reason: %d, MAC: %02X:%02X:%02X:%02X:%02X:%02X\r\n", (int) status, mac[0], mac[1], mac[2], mac[3],
+              mac[4], mac[5]);
+}
 
-    /****************************************************************************
-     * @fn  static void sl_wfx_ap_client_disconnected_callback(uint32_t status, uint8_t *mac)
-     * @brief
-     * Callback for AP client disconnect
-     * @param[in] status:
-     * @param[in]  mac:
-     * @return  None
-     *****************************************************************************/
-    static void sl_wfx_ap_client_disconnected_callback(uint32_t status, uint8_t * mac)
-    {
-        // TODO
-        EFR32_LOG("Client disconnected, reason: %d, MAC: %02X:%02X:%02X:%02X:%02X:%02X\r\n", (int) status, mac[0], mac[1], mac[2],
-                  mac[3], mac[4], mac[5]);
-    }
+/****************************************************************************
+ * @fn  static void sl_wfx_ap_client_disconnected_callback(uint32_t status, uint8_t *mac)
+ * @brief
+ * Callback for AP client disconnect
+ * @param[in] status:
```

Comment:
There does not seem to be any value in doccommenting parameters with empty content. Could we save some vertical space by not doing this? Also saying `return None` seems pointless when we see that the method is `void`.

Reply:
Thanks for input , I have made the changes according your comments.